content-services/content-sources-backend
# Fixes 3420: add service notify for templates

## Summary
Sends an event on a new kafka topic when a template is created, updated, or deleted. 

This has nothing to do with sending an event (notification) to the notifications service or getting emails. I've used the term "notification" in places because it is still technically a notification (we're notifying the patch service), I wasn't sure what else to call it, and there's an opportunity here to share code between the templates and notifications. I'm open to other names!

## Testing steps
1. In a terminal, `make kafka-topic-consume KAFKA_TOPIC=platform.content-sources.template`
2. Start the API server
3. Create a template. Check the terminal where you've just run the kafka consumer, you should see a notification.
4. Update a template. Check for a notification.
5. Delete a template. Check for a notification.
## Checklist
